### ISSUE TITLE
Filenames for attachments have no effect for downloaded files

### ISSUE THREAD

**Issue by staffanm** [jhpyle/docassemble#522]
After upgrading from 1.3.31 to 1.3.32, assembled Word or RTF files are always saved as file.docx/rtf, regardless of what has been specified in the attachment block. [This example from the documentation](https://docassemble.org/docs/documents.html#attachment) demonstrates the issue:

```yaml
mandatory: True
question: Your document is ready.
attachment:
  - name: A hello world document
    filename: Hello_World
    description: |
      A document with a classic message
    content: |
      Hello, world!
```

The resulting RTF file has the expected URL (eg https://example.org/uploadedfile/17/Hello_World.rtf) but when clicking the link, the file is downloaded as file.rdf -- i guess there is a `Content-Disposition: attachment; filename="file.rtf"` header? Could this be due to the upgraded dependecies in 1.3.32? I can't see anything obvious [in the diff](https://github.com/jhpyle/docassemble/compare/v1.3.31...v1.3.32).

**Comment by jhpyle** [comment]
Sorry about that. This was caused by the upgrade of `flask`. The issue should be fixed in 1.3.33 but let me know if you see any problems again.

**Comment by staffanm** [comment]
Thank you! I haven't seen any problems in the latest versions, so closing this.
